docs/src/development/sanitize-db.md

```diff
@@ -0,0 +1,139 @@
+1. Display all fields from your `user` table, to select which ones need to be redacted.
+   In a very basic database, this can be achieved with the `SELECT * FROM users;` MySQL query:
+
+   ```sql
+   MariaDB [main]> SELECT * FROM users;
+   +----+------------+---------------+---------------------------+---------------+
+   | ID | first_name | last_name     | user_email                | display_name  |
+   +----+------------+---------------+---------------------------+---------------+
+   |  1 | admin      | admin         | admin@yourcompany.com     | admin         |
+   |  2 | john       | doe           | john.doe@gmail.com        | john          |
+   |  3 | jane       | doe           | janedoe@ymail.com         | jane          |
+   +----+------------+---------------+---------------------------+---------------+
+   3 rows in set (0.00 sec)
+   ```
+
+1. Change the fields where PII Data is contained with the [`UPDATE` statement](https://dev.mysql.com/doc/refman/8.0/en/update.html).
+   For example, changing the first name of an user, when the email address is not using the company domain, would result in the following query:
+
+   ```sql
+   UPDATE user
+   SET first_name='redacted'
+   WHERE email NOT LIKE '%@yourcompany%'
+   ```
+
+   Adapt and run that query for all fields that you need to sanitized.
+   If you modify fields that you shouldn't alter,
+   [you can restore them](../administration/backup-and-restore.md#restore) from the dump you took in step 1.
+
+   You can create a script to automate the sanitization process to be run automatically on each new deployment.
+   Once you have a working script, add your script to sanitize the database to [a `deploy` hook](../user_guide/reference/platform-app-yaml.html#hooks):
+
+    ```yaml
+    deploy: |
+      cd /app/public
+      if [ "$PLATFORM_ENVIRONMENT_TYPE" = production ]; then
+        # Do whatever you want on the production site.
+      else
+        # The sanitization of the database should happen here (since it's non-production)
+        sanitize_the_database.sh
+      fi
+    ```
+
+<--->
+
+---
+title=With Drupal and Drush
+file=none
+highlight=false
+---
+
+1. To sanitize your database and get rid of sensitive, live information, use the `drush sql:sanitize` command.
+Add the sanitization of the database script in [your hook](../user_guide/reference/platform-app-yaml.html#hooks) for the non-production environment:
+
+  ```yaml
+  deploy: |
+    cd /app/public
+    if [ "$PLATFORM_ENVIRONMENT_TYPE" = production ]; then
+      # Do whatever you want on the production site.
+    else
+      drush -y sql:sanitize
+    fi
+    drush -y updatedb
+  ```
+
+More options are available. These are described in [Drush's documentation](https://www.drush.org/latest/commands/sql_sanitize/).
+
+{{< /codetabs >}}
+
+## What's next
+
+You learned how to remove sensitive data from a database.
+
+To replace sensitive data that with more meaningful data, you can add a `faker` to the process.
```

Comment:
```suggestion
To replace sensitive data that with other meaningful data, you can add a `faker` to the process.
```
A good formulation, but the faked data isn't _more_ meaningful than the original, sensitive data.